src/osm_configurator/view/states/state_manager.py

```diff
@@ -489,14 +499,14 @@ def freeze_state(self):
 
     def unfreeze_state(self):
         """
-        Unfreezes the state, by making all active frames interactable again and unlocking the state, allowing
+        Unfreezes the state, by making all active frames intractable again and unlocking the state, allowing
         state changes again!
         """
         # Unfreezing himself
         self._frozen: bool = False
 
         # Unfreezing all frames in the current state
-        # All Frames have to be unfreezable!
+        # All Frames have to be freezable!
```

Comment:
ok I see why you changed, mabye change to, all frames have to implement freezable

Reply:
ok